packages/components/src/components/input-field/increment-buttons.tsx

```diff
@@ -95,7 +99,9 @@ const IncrementButtons = ({
             </Button>
             <Button
                 id={`${id}_sub`}
-                className={'dc-input-wrapper__button dc-input-wrapper__button--decrement'}
+                className={`dc-input-wrapper__button dc-input-wrapper__button--decrement ${
+                    should_apply_disabled_style ? 'dc-input-wrapper__button--disabled' : ''
+                }`}
```

Comment:
```suggestion
className={classNames('dc-input-wrapper__button dc-input-wrapper__button--decrement', {
    'dc-input-wrapper__button--disabled': !!min_is_disabled,
})}
```
@kate-deriv we can get rid of should_apply_disabled_style. :)